android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/FhirCoreQuestionnaireFragment.kt

```diff
@@ -29,7 +30,13 @@ class FhirCoreQuestionnaireFragment : QuestionnaireFragment() {
         questionnaireItem.getExtensionByUrl(
             "https://fhir.labs.smartregister.org/barcode-type-widget-extension"
           )
-          .let { if (it == null) false else it.value.toString() == "barcode" }
+          .let { if (it == null) false else it.value.asStringValue() == "barcode" }
```

Comment:
It is possible to use a null safety let in this expression

Reply:
Okay, it's now much simpler